test-sbt/shared/src/main/scala/zio/test/sbt/ZTestEventHandlerSbt.scala

```diff
@@ -1,17 +1,40 @@
 package zio.test.sbt
 
-import sbt.testing.{EventHandler, TaskDef}
+import sbt.testing.{EventHandler, Status, TaskDef}
 import zio.{UIO, ZIO}
-import zio.test.{ExecutionEvent, ZTestEventHandler}
+import zio.test.{ExecutionEvent, TestAnnotation, TestFailure, ZTestEventHandler}
 
+/**
+ * Reports test results to SBT, ensuring that the `test` task fails if any ZIO
+ * test instances fail
+ *
+ * @param eventHandler
+ *   The underlying handler provided by SBT
+ * @param taskDef
+ *   The test task that we are reporting for
+ */
 class ZTestEventHandlerSbt(eventHandler: EventHandler, taskDef: TaskDef) extends ZTestEventHandler {
   def handle(event: ExecutionEvent): UIO[Unit] =
     event match {
       case evt @ ExecutionEvent.Test(_, _, _, _, _, _) =>
         ZIO.succeed(eventHandler.handle(ZTestEvent.convertEvent(evt, taskDef)))
-      case ExecutionEvent.SectionStart(_, _, _)      => ZIO.unit
-      case ExecutionEvent.SectionEnd(_, _, _)        => ZIO.unit
-      case ExecutionEvent.TopLevelFlush(_)           => ZIO.unit
-      case ExecutionEvent.RuntimeFailure(_, _, _, _) => ZIO.unit
```

Comment:
Surprise surprise, invoking `ZIO.unit` in the vent of a `RuntimeFailure` _won't_ give SBT any useful info. Who would've thought?